venom/src/main/java/com/github/venom/service/ServiceDelegate.kt

```diff
@@ -3,12 +3,17 @@ package com.github.venom.service
 import android.app.ActivityManager
 import android.content.Context
 import android.content.Intent
+import android.os.Build
 
 internal class ServiceDelegate(private val context: Context) {
 
     fun startService() {
         if (!isServiceRunning()) {
-            context.startService(Intent(context, VenomService::class.java))
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
I thinks It could be better to use  `ContextCompat.startForegroundService`

Reply:
This is much better indeed!